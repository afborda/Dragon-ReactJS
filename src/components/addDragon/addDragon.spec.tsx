import React from 'react'
import { render } from '@testing-library/react'

import AddDragon from '.'

describe('<Login/>', () => {
  it('should render correctly', () => {
    const { container } = render(<AddDragon />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
