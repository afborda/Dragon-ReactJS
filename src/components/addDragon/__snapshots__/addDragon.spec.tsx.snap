// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> should render correctly 1`] = `
<div
  class="style__CustomDiv-btqmCT jaKSvJ"
>
  <div
    class="style__Center-jZjGrQ ieRdVQ"
  >
    <div>
      <div>
        <div>
          <img
            alt="Image de perfil do usuario"
            class="style__CustomImg-gVgzYn iFHTNp"
            src="test-file-stub"
          />
        </div>
      </div>
      <form>
        <div>
          <input
            class="style__InputType-eWYwgd bgsYnK"
            placeholder="Nome do dragão"
            required=""
            type="text"
          />
          <input
            class="style__InputType-eWYwgd bgsYnK"
            placeholder="Tipo de dragão"
            required=""
            type="text"
          />
        </div>
        <div>
          <input
            class="style__Buttom-gGydxe bTXZgu"
            type="submit"
          />
        </div>
      </form>
    </div>
  </div>
</div>
`
